### PR TITLE
Qute: execute tag templates in isolation by default

### PR DESCRIPTION
**Pull request by mkouba**
- see https://github.com/quarkusio/quarkus/discussions/22285

This PR is a follows-up of https://github.com/quarkusio/quarkus/pull/22003.

### CODE REVIEW

File: docs/src/main/asciidoc/qute-reference.adoc [hunk 1]
```diff
@@ -1143,10 +1143,10 @@ Then, we can call the tag like this:
...
 <2> Tag content injected using the `nested-content` key in the tag template.
 
-By default, the tag template can reference data from the parent context.
-For example, the tag above could use the following expression `{items.size}`.
-However, sometimes it might be useful to disable this behavior and execute the tag as an _isolated_ template, i.e. without access to the context of the template that calls the tag.
-In this case, just add `_isolated` or `_isolated=true` argument to the call site, e.g. `{#itemDetail item showImage=true _isolated /}`.
+By default, a tag template cannot reference the data from the parent context.
+Qute executes the tag as an _isolated_ template, i.e. without access to the context of the template that calls the tag.
+However, sometimes it might be useful to change the default behavior and disable the isolation.
+In this case, just add `_isolated=false` to the call site, for example `{#itemDetail item showImage=true _isolated=false /}`.
```
Comment (ia3andy) [inline docs/src/main/asciidoc/qute-reference.adoc:1149]: I would use the opposite word: `_contextual=true` (or any other better word)

**Review by ia3andy — APPROVED** [review]
Looks good, just one small suggestion

File: docs/src/main/asciidoc/qute-reference.adoc (hunk 1 above)
Comment (mkouba) [reply docs/src/main/asciidoc/qute-reference.adoc:1149]: Hm, `contextual` is IMO too overloaded. `isolated` was already there but I didn't find a good antonym... In any case, we could add a "positive" alias. Maybe something like `_connected` or even just `_unisolated` (note that the `=true` part is optional)?

**Review by ia3andy — APPROVED** [review]
Thanks!

**Pull request merged**